nest-forms-backend/src/utils/types/prisma.ts

```diff
@@ -17,3 +17,12 @@ export type FormWithSelectedProperties = Prisma.FormsGetPayload<{
     formDefinitionSlug: true
   }
 }>
+
+export type EmailFormChecked = FormWithFiles & {
```

Comment:
Very nice! I had the same idea yesterday to implement this kind of logic all over the app. The `Form` is used in many stages and we only assume (and have no guarantees) that it is in a specific state/has specific properties in various consumers/methods etc.